commands/util.go

```diff
@@ -283,7 +283,7 @@ func printFullActivationList(activations []whisk.Activation) {
 func printActivationLogs(logs []string) {
 	for _, log := range logs {
 		if Flags.activation.strip {
-			fmt.Printf("%s\n", log[39:])
+			fmt.Printf("%s\n", log[strings.Index(log, ": ")+2:])
```

Comment:
or being completely timestamp format agnostic.. ready to ignore future timestamps with embedded ": "
```
log[strings.Index(log, "stdout: ")+strings.Index(log, "stderr: ")+9:]
```

Reply:
@mdeuser, would your example break if the user logs `stderr: ` and the pipe being used is `stdout: `, or vice versa?

Ex:
```
"2018-05-02T19:33:32.829992819Z stdout: this is stdout stderr: "
```

Reply:
good catch.. yeah.. my code example was too compact
```
skip_index := strings.Index(log, "stdout: ")
if skip_index == -1 {
  skip_index = strings.Index(log, "stdout: ")
}
fmt.Printf("%s\n", log[skip_index+8:])
```

Reply:
@mdeuser, I think that would result in the same type of problem.

https://play.golang.org/p/cTGf8yO4Krw

Reply:
https://play.golang.org/p/8zcL24npnPU

Reply:
https://play.golang.org/p/ffJ2Vo41S-6